lib/Db/AppointmentConfig.php

```diff
@@ -189,6 +189,10 @@ public function setAvailabilityAsArray(array $availability): self {
 		return $this;
 	}
 
+	public function getAvailabilityAsArray(): array {
+		return json_decode($this->getAvailability(), true);
```

Comment:
nitpick: use https://www.php.net/manual/en/json.constants.php#constant.json-throw-on-error so decoding errors do not go unnoticed.

Reply:
Done